var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -88,15 +94,17 @@ class Gromacs(CMakePackage):
     depends_on('plumed+mpi', when='+plumed+mpi')
     depends_on('plumed~mpi', when='+plumed~mpi')
     depends_on('fftw-api@3')
+    depends_on('mkl', when='fft=mkl')
```

Comment:
@amd-toolchain-support The new concretizer errors on the fact that `gromacs` has no `fft` variant.

Reply:
@alalazo we were referring old change when we raised a pull request. It has been fixed